.github/policies/resourceManagement.yml

```diff
@@ -24,6 +24,24 @@ configuration:
       - closeIssue
       - addReply:
           reply: "Closing this issue as we've seen no reply to the request for more information. If you are able to get the requested information, please add it to the issue and we will retriage it. "
+   
+    - description: Close stale PRs
+      frequencies:
+      - daily:
+          hour: 7
+      - daily:
+          hour: 19
+      filters:
+      - isOpen
+      - isPullRequest
+      - not:
+          isDraft
+      - noActivitySince:
+          days: 14
+      actions:
+      - closePullRequest

```

Comment:
Can we have this switch things back to draft instead? Or label the PRs closed this way? My concern being if somebody is out for a bit. it'll be harder for them to figure out which things got auto-closed when they come back.

Reply:
@CyrusNajmabadi ^

Reply:
Is it possible to exempt the Roslyn team?

Reply:
No.  Let's just move to closed after a month.  If you're out a month, you should figure out what to do with your open PRs anyways.

Reply:
I would not exempt the team.  We're both a large source of this already, and we should be able to complete things within a month, or take them out if we're making no progress on them.